src/App.jsx

```diff
@@ -1,20 +1,13 @@
-import { useState, useEffect } from 'react';
-
-
-function App() {
-  const [unidades, setUnidades] = useState([])
-
-  useEffect(()=>{
-    fetch("http://localhost:3003/unidades")
-      .then((resposta)=>{return resposta.json()})
-      .then((conteudo)=>{setUnidades(conteudo)})
-  },[])
-    console.log(unidades)
-  return (
-    <>
-      Esta funcionando
-    </>
-  )
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
```

Comment:
Marcio o Navigate não foi usado, que tal remover?
```suggestion
import { BrowserRouter, Routes, Route } from "react-router-dom";
```